packages/react-utilities/src/compose/getSlots.ts

```diff
@@ -68,9 +67,9 @@ export function getSlots<R extends SlotPropsRecord>(
 function getSlot<R extends SlotPropsRecord, K extends keyof R>(
   state: ComponentState<R>,
   slotName: K,
-): readonly [React.ElementType<R[K]>, R[K]] {
+): readonly [React.ElementType<R[K]> | null, R[K]] {
```

Comment:
Shouldn't this return `|null` only for nullable slots?

Reply:
`getSlot` function is only internally used by `getSlots`, it doesn't have enough types information to distinguish, the `null` only for nullable slots is treated by `getSlots` method